tabpfn_client/prompt_agent.py

```diff
@@ -207,6 +208,34 @@ def prompt_reusing_existing_token(cls):
 
         print(cls.indent(prompt))
 
+    @classmethod
+    def reverify_email(cls, user_auth_handler: "UserAuthenticationClient"):
+        prompt = "\n".join(
+            [
+                "Please check your inbox for the verification email.",
+                "Note: The email might be in your spam folder or could have expired.",
+            ]
+        )
+        print(cls.indent(prompt))
+        retry_verification = "\n".join(
+            [
+                "Do you want to resend email verification link? (y/n): ",
+            ]
+        )
+        choice = cls._choice_with_retries(retry_verification, ["y", "n"])
+        if choice == "y":
+            email = input(cls.indent("Please enter your email: "))
```

Comment:
please don't ask the user for the email address and password here, but get it from our server or save it here. This is very error prone.

Reply:
Okayy sure!

Reply:
I am using the existing token : from which i will use the get_current_user directly on server side.